src/Modules/DisplayTask/components/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ChangeEvent } from "react";
+import { NavLink } from "react-router-dom";
+
+import Loader from "../../../Shared/components/Loader";
+
+import { ToDoData } from "../../../data/types/types";
+import { ROUTES } from "../../../appContants";
+import { TASK_STATE } from "../../../constant";
+
+interface IProps {
+  data: ToDoData;
+  loading: boolean;
+  error: string;
+  markTodoCompleted: (e: ChangeEvent<HTMLSelectElement>) => void;
+  modalHandler: () => void;
+}
+
+const DisplayTaskComponent: React.FC<IProps> = (props) => {
+  const {
+    data,
+    loading,
+    error,
+    markTodoCompleted,
+    modalHandler,
+  } = props;
+  return (
+    <>
+      {loading ? (
```

Comment:
move this loading condition outside of return 
like 
if(loading){
     return <Loader/>
}

Reply:
Done